docs/src/main/sphinx/connector/prometheus.rst

```diff
@@ -47,18 +47,26 @@ Configuration properties
 
 The following configuration properties are available:
 
-======================================== ============================================================================================
-Property name                                   Description
-======================================== ============================================================================================
-``prometheus.uri``                       Where to find Prometheus coordinator host
-``prometheus.query.chunk.size.duration`` The duration of each query to Prometheus
-``prometheus.max.query.range.duration``  Width of overall query to Prometheus, will be divided into query-chunk-size-duration queries
-``prometheus.cache.ttl``                 How long values from this config file are cached
-``prometheus.auth.user``                 Username for basic authentication
-``prometheus.auth.password``             Password for basic authentication
-``prometheus.bearer.token.file``         File holding bearer token if needed for access to Prometheus
-``prometheus.read-timeout``              How much time a query to Prometheus has before timing out
-======================================== ============================================================================================
+============================================= ============================================================================================
+Property name                                 Description
+============================================= ============================================================================================
+``prometheus.uri``                            Where to find Prometheus coordinator host
+``prometheus.query.chunk.size.duration``      The duration of each query to Prometheus
+``prometheus.max.query.range.duration``       Width of overall query to Prometheus, will be divided into query-chunk-size-duration queries
+``prometheus.cache.ttl``                      How long values from this config file are cached
+``prometheus.auth.user``                      Username for basic authentication
+``prometheus.auth.password``                  Password for basic authentication
+``prometheus.bearer.token.file``              File holding bearer token if needed for access to Prometheus
+``prometheus.read-timeout``                   How much time a query to Prometheus has before timing out
+``prometheus.case-insensitive-name-matching`` Match Prometheus metric names case insensitively.
```

Comment:
Let's append `. Defaults to ``false``.` and remove following changes between L63 and L69.

Reply:
Done, thx~

Reply:
Done, thx~